### ISSUE TITLE
Don't install `tests` as a package

### ISSUE THREAD

**Issue by PeterJCLaw** [TylerYep/flake8-future-annotations#11]
Currently the `setup.cfg` is configured with `packages = find:` without excluding this project's `tests` folder. This means that a fresh install of this package creates two directories within the `site-packages` folder of the python installation: `flake8_future_annotations` and `tests`. The latter is potentially confusing and probably undesirable.

While I do see there is benefit to including the tests in the distributed source tarball, having them actually installed into consumers' environments doesn't feel like such a good idea.

**Comment by TylerYep** [comment]
Yep, that's a mistake with the packaging setup. Will accept a PR fixing this!
